src/pages/Version.tsx

```diff
@@ -117,9 +124,11 @@ export const VersionPage: React.FC = () => {
   } = patch || {};
   const isPatchOnCommitQueue = commitQueuePosition !== null;
 
-  const title = isPatch
-    ? `Patch - ${patchNumber}`
-    : `Version - ${revision?.substr(0, 6)}`;
+  // If a revision exists
+  const versionText = revision?.length
+    ? revision?.substr(0, 7)
```

Comment:
we should use substring instead of substr: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#the_difference_between_substring_and_substr